_pages/dev/performance/above-the-fold.md

```diff
@@ -56,17 +56,17 @@ ProductDetailsPageTemplate: {
 
 ## CSS Configuration
 
-By default, when page slots are loaded on the page, there's no minimum height of page slots or components available. Page slots will initially all have no height, which brings them all in the view port. This eliminates the concept of deferred loading, which is based on content not being the viewport.
+By default, when page slots are loaded on the page, there is no minimum height available for the page slots or components. All page slots initially have no height, which brings them all into the viewport. This prevents the deferred loading technique from working, because it depends on content not being in the viewport.
 
-Given that content can be added at runtime, it is not possible to implement a (hard-coded) minimum heights for page slots or components; it all depends on what the business will add at runtime.
+Given that content can be added at runtime, it is not possible to implement a (hard-coded) minimum height for page slots or components – it all depends on what the business will add at runtime.
 
-While this lack of minimum height could be filled up by so-called _ghost design_ CSS rules, there will always be a gap between the ghost design and the actual content.
+While this lack of minimum height could be filled up by so-called "ghost design" CSS rules, there will always be a gap between the ghost design and the actual content.
```

Comment:
can we add the following to the end. 
Moreover, "ghost design" rules is taking an implementation effort and might not be available.

Reply:
Done.